behat.yml.dist

```diff
@@ -1,4 +1 @@
 default:
-  gherkin:
```

Comment:
Without this config, this file is not actually needed but I thought it would be good to keep it as a reference for anyone looking at the code and trying to find where we configure things